lib/collector.class.js

```diff
@@ -7,9 +7,9 @@ const COLLECT_TIMEOUT = 'Metasync: Collector timed out';
 const COLLECT_CANCELED = 'Metasync: Collector cancelled';
 
 class Collector {
-  constructor(
-    expected // number or array of string, count or keys
-  ) {
+  // Collector constructor
+  //   expected - <number> | <string[]>
```

Comment:
You shouldn't have removed the explanation for the argument:
```suggestion
  //   expected - <number> | <string[]>, count or keys
```